src/main/kotlin/net/atos/zac/app/task/TaskRestService.kt

```diff
@@ -230,12 +230,12 @@ class TaskRestService @Inject constructor(
         }
         taakVariabelenService.setTaskData(updatedTask, restTask.taakdata)
         taakVariabelenService.setTaskinformation(updatedTask, restTask.taakinformatie)
-        flowableTaskService.completeTask(updatedTask).let {
+        val completedTask = flowableTaskService.completeTask(updatedTask).also {
             indexeerService.addOrUpdateZaak(restTask.zaakUuid, false)
             eventingService.send(ScreenEventType.TAAK.updated(it))
             eventingService.send(ScreenEventType.ZAAK_TAKEN.updated(restTask.zaakUuid))
-            return restTaskConverter.convert(it)
```

Comment:
WHY WAS THIS EVEN POSSIBLE 😱

Reply:
I should not have changed this code in this spike as this code will probably never be merged..